# SOME DESCRIPTIVE TITLE.
# Copyright (C) YEAR THE PACKAGE'S COPYRIGHT HOLDER
# This file is distributed under the same license as the PACKAGE package.
# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
# 
# Translators:
# Kheyyam Gojayev <xxmn77@gmail.com>, 2022
# 
#, fuzzy
msgid ""
msgstr ""
"Project-Id-Version: PACKAGE VERSION\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2022-05-27 11:18+0200\n"
"PO-Revision-Date: 2017-08-09 10:34+0000\n"
"Last-Translator: Kheyyam Gojayev <xxmn77@gmail.com>, 2022\n"
"Language-Team: Azerbaijani (https://www.transifex.com/calamares/teams/20061/az/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"Language: az\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#: src/modules/packages/main.py:54 src/modules/packages/main.py:65
#: src/modules/packages/main.py:75
msgid "Install packages."
msgstr "Paketləri quraşdırmaq."

#: src/modules/packages/main.py:63
#, python-format
msgid "Processing packages (%(count)d / %(total)d)"
msgstr "(%(count)d / %(total)d) paketləri işlənir"

#: src/modules/packages/main.py:68
#, python-format
msgid "Installing one package."
msgid_plural "Installing %(num)d packages."
msgstr[0] "Bir paket quraşdırılır."
msgstr[1] "%(num)d paket quraşdırılır."

#: src/modules/packages/main.py:71
#, python-format
msgid "Removing one package."
msgid_plural "Removing %(num)d packages."
msgstr[0] "Bir paket silinir"
msgstr[1] "%(num)d paket silinir."

#: src/modules/packages/main.py:725 src/modules/packages/main.py:737
#: src/modules/packages/main.py:765
msgid "Package Manager error"
msgstr "Paket meneceri xətası"

#: src/modules/packages/main.py:726
msgid ""
"The package manager could not prepare updates. The command <pre>{!s}</pre> "
"returned error code {!s}."
msgstr ""
"Bu paket meneceri yenilənmələri hazırlaya bilmədi. <pre>{!s}</pre> əmri xəta"
" kodu {!s} ilə cavab verdi."

#: src/modules/packages/main.py:738
msgid ""
"The package manager could not update the system. The command <pre>{!s}</pre>"
" returned error code {!s}."
msgstr ""
"Paket meneceri sistemi yeniləyə bimədi. <pre>{!s}</pre> əmri xəta kodu {!s} "
"ilə cavab verdi."

#: src/modules/packages/main.py:766
msgid ""
"The package manager could not make changes to the installed system. The "
"command <pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Paket meneceri dəyişiklikləri sistemə tətbiq edə bilmədi. <pre>{!s}</pre> "
"əmri xəta kodu {!s} ilə cavab verdi."

#: src/modules/networkcfg/main.py:29
msgid "Saving network configuration."
msgstr "Şəbəkə ayarları saxlanılır."

#: src/modules/networkcfg/main.py:105 src/modules/initcpiocfg/main.py:235
#: src/modules/initcpiocfg/main.py:239 src/modules/openrcdmcryptcfg/main.py:72
#: src/modules/openrcdmcryptcfg/main.py:76 src/modules/localecfg/main.py:140
#: src/modules/mount/main.py:229 src/modules/fstab/main.py:394
#: src/modules/fstab/main.py:400 src/modules/fstab/main.py:428
#: src/modules/initramfscfg/main.py:85 src/modules/initramfscfg/main.py:89
#: src/modules/rawfs/main.py:164
msgid "Configuration Error"
msgstr "Tənzimləmə xətası"

#: src/modules/networkcfg/main.py:106 src/modules/initcpiocfg/main.py:240
#: src/modules/openrcdmcryptcfg/main.py:77 src/modules/localecfg/main.py:141
#: src/modules/fstab/main.py:401 src/modules/initramfscfg/main.py:90
msgid "No root mount point is given for <pre>{!s}</pre> to use."
msgstr ""
"<pre>{!s}</pre> istifadə etmək üçün kök qoşulma nöqtəsi təyin edilməyib."

#: src/modules/initcpiocfg/main.py:28
msgid "Configuring mkinitcpio."
msgstr "mkinitcpio tənzimlənir."

#: src/modules/initcpiocfg/main.py:236 src/modules/openrcdmcryptcfg/main.py:73
#: src/modules/mount/main.py:230 src/modules/fstab/main.py:395
#: src/modules/initramfscfg/main.py:86 src/modules/rawfs/main.py:165
msgid "No partitions are defined for <pre>{!s}</pre> to use."
msgstr "<pre>{!s}</pre> istifadə etmək üçün bölmələr təyin edilməyib"

#: src/modules/mkinitfs/main.py:27
msgid "Creating initramfs with mkinitfs."
msgstr "mkinitfs ilə initramfs yaradılır"

#: src/modules/mkinitfs/main.py:49
msgid "Failed to run mkinitfs on the target"
msgstr "Hədəfdə mkinitfs başlatmaq baş tutmadı"

#: src/modules/mkinitfs/main.py:50 src/modules/dracut/main.py:50
msgid "The exit code was {}"
msgstr "Çıxış kodu {} idi"

#: src/modules/openrcdmcryptcfg/main.py:26
msgid "Configuring OpenRC dmcrypt service."
msgstr "OpenRC dmcrypt xidməti tənzimlənir."

#: src/modules/unpackfs/main.py:34
msgid "Filling up filesystems."
msgstr "Fayl sistemlərini doldurmaq."

#: src/modules/unpackfs/main.py:254
msgid "rsync failed with error code {}."
msgstr "rsync uğursuz oldu, xəta kodu: {}."

#: src/modules/unpackfs/main.py:299
msgid "Unpacking image {}/{}, file {}/{}"
msgstr ""
"Tərkibi çıxarılan quraşdırma faylı - image {}/{}, çıxarılan faylların sayı "
"{}/{}"

#: src/modules/unpackfs/main.py:314
msgid "Starting to unpack {}"
msgstr "Tərkiblərini açmağa başladılır {}"

#: src/modules/unpackfs/main.py:323 src/modules/unpackfs/main.py:467
msgid "Failed to unpack image \"{}\""
msgstr "\"{}\" quraşdırma faylının tərkibini çıxarmaq alınmadı"

#: src/modules/unpackfs/main.py:430
msgid "No mount point for root partition"
msgstr "Kök bölməsi üçün qoşulma nöqtəsi yoxdur"

#: src/modules/unpackfs/main.py:431
msgid "globalstorage does not contain a \"rootMountPoint\" key."
msgstr "globalstorage tərkibində bir \"rootMountPoint\" açarı yoxdur."

#: src/modules/unpackfs/main.py:434
msgid "Bad mount point for root partition"
msgstr "Kök bölməsi üçün xətalı qoşulma nöqtəsi"

#: src/modules/unpackfs/main.py:435
msgid "rootMountPoint is \"{}\", which does not exist."
msgstr "rootMountPoint \"{}\" mövcud deyil."

#: src/modules/unpackfs/main.py:439 src/modules/unpackfs/main.py:455
#: src/modules/unpackfs/main.py:459 src/modules/unpackfs/main.py:465
#: src/modules/unpackfs/main.py:480
msgid "Bad unpackfs configuration"
msgstr "Xətalı unpackfs tənzimləməsi"

#: src/modules/unpackfs/main.py:440
msgid "There is no configuration information."
msgstr "Tənzimləmə məlumatı yoxdur"

#: src/modules/unpackfs/main.py:456
msgid "The filesystem for \"{}\" ({}) is not supported by your current kernel"
msgstr "\"{}\" ({}) fayl sistemi sizin nüvəniz tərəfindən dəstəklənmir"

#: src/modules/unpackfs/main.py:460
msgid "The source filesystem \"{}\" does not exist"
msgstr "\"{}\" mənbə fayl sistemi mövcud deyil"

#: src/modules/unpackfs/main.py:466
msgid ""
"Failed to find unsquashfs, make sure you have the squashfs-tools package "
"installed."
msgstr ""
"Unsquashfs tapılmadı, squashfs-tools paketinin quraşdırıldığına əmin olun."

#: src/modules/unpackfs/main.py:481
msgid "The destination \"{}\" in the target system is not a directory"
msgstr "Hədəf sistemində təyin edilən \"{}\", qovluq deyil"

#: src/modules/services-systemd/main.py:26
msgid "Configure systemd services"
msgstr "Systemd xidmətini tənzimləmək"

#: src/modules/services-systemd/main.py:59
#: src/modules/services-openrc/main.py:93
msgid "Cannot modify service"
msgstr "Xidmətdə dəyişiklik etmək mümkün olmadı"

#: src/modules/services-systemd/main.py:60
msgid ""
"<code>systemctl {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>systemctl {arg!s}</code> chroot çağırışına xəta kodu ilə cavab verdi "
"{num!s}."

#: src/modules/services-systemd/main.py:63
#: src/modules/services-systemd/main.py:69
msgid "Cannot enable systemd service <code>{name!s}</code>."
msgstr "<code>{name!s}</code> systemd xidməti aktiv edilmədi."

#: src/modules/services-systemd/main.py:65
msgid "Cannot enable systemd target <code>{name!s}</code>."
msgstr "<code>{name!s}</code> systemd hədəfi aktiv edilmədi"

#: src/modules/services-systemd/main.py:67
msgid "Cannot enable systemd timer <code>{name!s}</code>."
msgstr "<code>{name!s}</code> systemd taymeri aktiv edilə bilmir."

#: src/modules/services-systemd/main.py:71
msgid "Cannot disable systemd target <code>{name!s}</code>."
msgstr "<code>{name!s}</code> systemd hədfi sönsürülmədi."

#: src/modules/services-systemd/main.py:73
msgid "Cannot mask systemd unit <code>{name!s}</code>."
msgstr "<code>{name!s}</code> systemd vahidi maskalanmır."

#: src/modules/services-systemd/main.py:75
msgid ""
"Unknown systemd commands <code>{command!s}</code> and "
"<code>{suffix!s}</code> for unit {name!s}."
msgstr ""
"Naməlum systemd əmrləri <code>{command!s}</code> və <code>{suffix!s}</code> "
"{name!s} vahidi üçün."

#: src/modules/dummypython/main.py:35
msgid "Dummy python job."
msgstr "Dummy python işi."

#: src/modules/dummypython/main.py:37 src/modules/dummypython/main.py:93
#: src/modules/dummypython/main.py:94
msgid "Dummy python step {}"
msgstr "{} Dummy python addımı"

#: src/modules/bootloader/main.py:43
msgid "Install bootloader."
msgstr "Önyükləyici qurulur."

#: src/modules/bootloader/main.py:614
msgid "Failed to install grub, no partitions defined in global storage"
msgstr "Grub quraşdırılmadı, ümumi yaddaş üçün bölmələr təyin olunmayıb"

#: src/modules/bootloader/main.py:782
msgid "Bootloader installation error"
msgstr "Önyükləyicinin quraşdırılmasında xəta"

#: src/modules/bootloader/main.py:783
msgid ""
"The bootloader could not be installed. The installation command "
"<pre>{!s}</pre> returned error code {!s}."
msgstr ""
"Önyükləyici quraşdırıla bilmədi. Quraşdırma əmri <pre>{!s}</pre>, xəta kodu "
"{!s} ilə cavab verdi."

#: src/modules/localecfg/main.py:31
msgid "Configuring locales."
msgstr "Lokallaşma tənzimlənir."

#: src/modules/mount/main.py:42
msgid "Mounting partitions."
msgstr "Disk bölmələri qoşulur."

#: src/modules/mount/main.py:88 src/modules/mount/main.py:124
msgid "Internal error mounting zfs datasets"
msgstr "Zfs verilənlər dəstinin qoşulmasında daxil xəta"

#: src/modules/mount/main.py:100
msgid "Failed to import zpool"
msgstr "Zpool idxalı baş tutmadı"

#: src/modules/mount/main.py:116
msgid "Failed to unlock zpool"
msgstr "Zpool kiliddən çıxarıla bilmədi"

#: src/modules/mount/main.py:133 src/modules/mount/main.py:138
msgid "Failed to set zfs mountpoint"
msgstr "Zfs qoşulma nöqtəsi təyin olunmadı"

#: src/modules/mount/main.py:253
msgid "zfs mounting error"
msgstr "zfs qoşulmasında xəta"

#: src/modules/plymouthcfg/main.py:27
msgid "Configure Plymouth theme"
msgstr "Plymouth mövzusu tənzimlənməsi"

#: src/modules/fstab/main.py:28
msgid "Writing fstab."
msgstr "fstab yazılır."

#: src/modules/fstab/main.py:429
msgid "No <pre>{!s}</pre> configuration is given for <pre>{!s}</pre> to use."
msgstr ""
"İstifadə etmək üçün, <pre>{!s}</pre> tənzimləməsi, <pre>{!s}</pre> üçün "
"göstərilməyib."

#: src/modules/services-openrc/main.py:29
msgid "Configure OpenRC services"
msgstr "OpenRC xidmətlərini tənzimləmək"

#: src/modules/services-openrc/main.py:57
msgid "Cannot add service {name!s} to run-level {level!s}."
msgstr "{name!s} xidməti {level!s} işləmə səviyyəsinə əlavə edilə bilmir."

#: src/modules/services-openrc/main.py:59
msgid "Cannot remove service {name!s} from run-level {level!s}."
msgstr "{name!s} xidməti {level!s} iş səviyyəsindən silinə bilmir."

#: src/modules/services-openrc/main.py:61
msgid ""
"Unknown service-action <code>{arg!s}</code> for service {name!s} in run-"
"level {level!s}."
msgstr ""
"{level!s} işləmə səviyyəsindəki {name!s} xidməti üçün naməlum "
"<code>{arg!s}</code> xidmət fəaliyyəti."

#: src/modules/services-openrc/main.py:94
msgid ""
"<code>rc-update {arg!s}</code> call in chroot returned error code {num!s}."
msgstr ""
"<code>rc-update {arg!s}</code> chroot-da çağırışına {num!s} xəta kodu ilə "
"cavab verildi."

#: src/modules/services-openrc/main.py:101
msgid "Target runlevel does not exist"
msgstr "Hədəf işləmə səviyyəsi mövcud deyil"

#: src/modules/services-openrc/main.py:102
msgid ""
"The path for runlevel {level!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr ""
"{level!s} işləmə səviyyəsi üçün <code>{path!s}</code> yolu mövcud deyil."

#: src/modules/services-openrc/main.py:110
msgid "Target service does not exist"
msgstr "Hədəf xidməti mövcud deyil"

#: src/modules/services-openrc/main.py:111
msgid ""
"The path for service {name!s} is <code>{path!s}</code>, which does not "
"exist."
msgstr "{name!s} üçün <code>{path!s}</code> yolu mövcud deyil."

#: src/modules/dracut/main.py:27
msgid "Creating initramfs with dracut."
msgstr "Dracut ilə initramfs yaratmaq."

#: src/modules/dracut/main.py:49
msgid "Failed to run dracut on the target"
msgstr "Hədəfdə dracut başladılmadı"

#: src/modules/grubcfg/main.py:28
msgid "Configure GRUB."
msgstr "GRUB tənzimləmələri"

#: src/modules/displaymanager/main.py:524
msgid "Cannot write KDM configuration file"
msgstr "KDM tənzimləmə faylı yazıla bilmir"

#: src/modules/displaymanager/main.py:525
msgid "KDM config file {!s} does not exist"
msgstr "KDM tənzimləmə faylı {!s} mövcud deyil"

#: src/modules/displaymanager/main.py:586
msgid "Cannot write LXDM configuration file"
msgstr "LXDM tənzimləmə faylı yazıla bilmir"

#: src/modules/displaymanager/main.py:587
msgid "LXDM config file {!s} does not exist"
msgstr "LXDM tənzimləmə faylı {!s} mövcud deyil"

#: src/modules/displaymanager/main.py:670
msgid "Cannot write LightDM configuration file"
msgstr "LightDM tənzimləmə faylı yazıla bilmir"

#: src/modules/displaymanager/main.py:671
msgid "LightDM config file {!s} does not exist"
msgstr "LightDM tənzimləmə faylı {!s} mövcud deyil"

#: src/modules/displaymanager/main.py:745
msgid "Cannot configure LightDM"
msgstr "LightDM tənzimlənə bilmir"

#: src/modules/displaymanager/main.py:746
msgid "No LightDM greeter installed."
msgstr "LightDM qarşılama quraşdırılmayıb."

#: src/modules/displaymanager/main.py:777
msgid "Cannot write SLIM configuration file"
msgstr "SLİM tənzimləmə faylı yazıla bilmir"

#: src/modules/displaymanager/main.py:778
msgid "SLIM config file {!s} does not exist"
msgstr "SLİM tənzimləmə faylı {!s} mövcud deyil"

#: src/modules/displaymanager/main.py:992
msgid "No display managers selected for the displaymanager module."
msgstr "displaymanager modulu üçün ekran menecerləri seçilməyib."

#: src/modules/displaymanager/main.py:993
msgid ""
"The displaymanagers list is empty or undefined in both globalstorage and "
"displaymanager.conf."
msgstr ""
"Ekran menecerləri siyahısı həm qlobal yaddaşda, həm də displaymanager.conf-"
"da boşdur və ya təyin olunmamışdır."

#: src/modules/displaymanager/main.py:1075
msgid "Display manager configuration was incomplete"
msgstr "Ekran meneceri tənzimləmələri başa çatmadı"

#: src/modules/initramfscfg/main.py:32
msgid "Configuring initramfs."
msgstr "initramfs tənzimlənir."

#: src/modules/hwclock/main.py:26
msgid "Setting hardware clock."
msgstr "Aparat saatını ayarlamaq."

#: src/modules/rawfs/main.py:26
msgid "Installing data."
msgstr "Quraşdırılma tarixi."
